app/src/components/pages/collectionsPage/collectionsPage.tsx

```diff
@@ -37,20 +143,111 @@ export const CollectionsPage = ({ data }) => {
           subtitle="Explore the New York Public Library's diverse collections, including digitized photographs, manuscripts, maps, and more. Start exploring by using the search bar below or browse through the collections."
         />
         <SearchBar
-          sx={{ maxWidth: "462px", marginTop: "l" }}
+          sx={{ maxWidth: "462px" }}
           id={"search-collections"}
           textInputProps={{
             isClearable: true,
+            isClearableCallback: () => {
+              setcurrentCollectionKeywords(DEFAULT_SEARCH_TERM);
+            },
             labelText: "Search by collection title",
-            name: "searchinput",
+            name: "collection_keywords",
             placeholder: "Search by collection title",
+            defaultValue: currentCollectionKeywords,
+            onChange: (e) => handleSearchChange(e),
           }}
-          onSubmit={function (event: React.FormEvent): void {}}
+          onSubmit={handleSearchSubmit}
           labelText={""}
         />
       </Box>
       <HorizontalRule sx={{ marginTop: "xxl", marginBottom: "xxl" }} />
-      <SearchResults showFilter isSearchPage={false} data={data} />
+      <Box
+        sx={
+          collections?.length > 0
+            ? { display: "flex", gap: "xs", marginBottom: "l" }
+            : { display: "none", gap: "xs", marginBottom: "l" }
+        }
+      >
+        <Text sx={{ fontWeight: "500", marginBottom: 0, marginTop: "xs" }}>
+          {" "}
+          Sort by{" "}
+        </Text>{" "}
+        <Menu
+          showSelectionAsLabel
+          showLabel
+          selectedItem={currentSort}
+          labelText={"Sort By"}
+          listItemsData={[
+            {
+              id: "date-desc",
+              label: "Newest to oldest",
+              onClick: onMenuClick,
+              type: "action",
+            },
+            {
+              id: "date-asc",
+              label: "Oldest to newest",
+              onClick: onMenuClick,
+              type: "action",
+            },
+            {
+              id: "title-asc",
+              label: "Title A to Z",
+              onClick: onMenuClick,
+              type: "action",
+            },
+            {
+              id: "title-desc",
+              label: "Title Z to A",
+              onClick: onMenuClick,
+              type: "action",
+            },
+          ]}
+        />
+      </Box>
+      <Heading
+        size="heading5"
+        sx={
+          collections?.length > 0
+            ? { marginBottom: "l" }
+            : { display: "none", marginBottom: "l" }
+        }
+        ref={headingRef}
+        tabIndex={-1}
+        id="collections"
```

Comment:
I'm seeing for searches or sorting updates with many results that the page jumps up and the header blocks the content:

<img width="413" alt="Screenshot 2024-12-13 at 12 35 15 PM" src="https://github.com/user-attachments/assets/9f8dd2c5-13e9-402b-babf-95bc66979ad4" />

Try replicating with a blank search and then updating the sort value. This is a problem but one you can either tackle later or remove the `#collections` from the URL. I think you can remove it -- it might be redundant since you are programmatically focusing on the text after user actions.

Reply:
lana and lily asked for the #collections to be added to the url so the pagination behavior is consistent with the divisions landing page.

Reply:
Good point. In this case, it's not great user experience for the header to block the page in this manner. You can't see the searchbar so I think the sort menu should not be visible at all rather than partially covered. I think you can manage to push the page around when the element is focused like this and it might already be in the header for this app. We can ask Emma next week.

Reply:
It's not great user experience for now but I don't believe the header is going to block the menu after we move the menu to the right side https://newyorkpubliclibrary.atlassian.net/browse/DR-3323.

Reply:
I think we should wait until DR-3323 is done first.

Reply:
this is an interesting question because to me, intuitively, on clicking "Search" the page shouldn't pull you to the "Displaying results", it should just keep you where you are scroll-wise, but on pagination, it should. Just me ?
The scroll padding on the header is very statically set right now (https://github.com/NYPL/digital-collections/blob/c34939a29a15c1e88f11d8e799f5d37a5afde2e4/app/globals.css) and doesn't have anything to do with where focus is, only with how tall the header is, so a more dynamic approach would be needed if we want to conditionally add padding to where the header ends up when using anchor links

Reply:
I don't think we'll have to conditionally add padding after the ticket mentioned above is done, which is a feature that will block the release of this page but can be worked on separately

Reply:
Regardless of whether the header blocks the menu, the jump down the page on Search is a weird experience– is there a way to focus on the "Displaying results" without scrolling down the page, unless "Displaying results" isn't already visible?

Reply:
The behavior on Search is a good point. I'm going to check in with Lana and Lily about this because it is technically possible to _not_ scroll down to the "Displaying results" section on Search by not add `#collections` to the url after search is hit. Not the most fun thing to implement but it is feasible.

Reply:
Okay yeah if I'm the only person that feels this way then dw about it but it does just seem unintuitive to me that the screen scrolls even though the element is already visible

Reply:
Lily and Lana are on board with removing it from Search - I also think it's weird adding it Sort so I removed it from both lol

Reply:
kk added it back to sort as per lanas request and moved sort to the right so the menu isn't blocked off by the header.